pom.xml

```diff
@@ -379,7 +379,7 @@
                 <plugin>
                     <groupId>org.sonatype.plugins</groupId>
                     <artifactId>nexus-staging-maven-plugin</artifactId>
-                    <version>1.6.13</version> <!-- 1.6.9 and 1.6.10 are completely broken!! -->
+                    <version>1.7.0</version> <!-- 1.6.9 and 1.6.10 are completely broken!! -->
                     <!-- Perhaps try 1.6.12? -->
```

Comment:
@belaban Let's remove these stale comments then - PR is up https://github.com/belaban/JGroups/pull/796